client/src/app/services/hooks/use-manage-play.ts

```diff
@@ -141,6 +148,7 @@ const useManagePlay = (): ManagePlayData => {
 				loading: false,
 			});
 		}
+		closeDeploysWsConnection();
```

Comment:
Shouldn't we close the connection at the beginning of the method to avoid the delay introduced by the concurrent processing of the code above from React?